### ISSUE TITLE
When consolidating, settxfee does not behave as expected.

### ISSUE THREAD

**Issue by mverrilli** [decred/dcrwallet#787]
I set the fee using: 

dcrctl --wallet settxfee 0.0001

At this point I'll just mentioned, that yes... this is too low. I didn't realize it at the time and had an extra 0.  However, the behavior still isn't right. 

Then I'll attempt to do a consolidation on a lot of transactions and I'll get an error like: 

xxxxx has 750000 fees which is under the required amount of 6353700

So then I'll do a little math to find the minimum... 6353700 / 750000 * 0.0001 = 0.00084716

And I'll set my txfee to that number (or just very slightly over... rounding issue I assume) and it'll work. When viewing the actual transaction, it is at .001/kb. 

Each time, the calculation and set txfee is wildly different. Over 4 consolidations, I had the following work: 0.00023, 0.00033, .000484497, 0.00084719... each one ended up with a transaction with a fee at 0.001/kb.

So next I tried a smaller consolidate using 0.001.  It ended up charging 0.002000 fee for the transaction on a 1368 byte transaction. (0.00146199/kb). 

May be related to #786

**Comment by jrick** [comment]
Yes they're related.  The fee is calculated based on the expected size which will always overshoot the actual size (or else there would be rejected txs due to some actual fees being too low).  Because that size estimation code is producing much larger estimated sizes than it should be, the transactions end up with higher relative fee.

**Comment by alexlyp** [comment]
Should be fixes in #788

**Comment by mverrilli** [comment]
@alexlyp Just FYI I tested master and I still have the issue described.

**Comment by alexlyp** [comment]
ok, reopening.  i'll take a quick look
